test/runtests.jl

```diff
@@ -18,67 +20,211 @@ import Impute:
+                Impute.dropobs!(a2; context=ctx)
+                @test a2 == expected
+            end
+
+            @testset "Matrix" begin
+                # Because we're removing 2 of our 5 rows we need to change the limit.
+                ctx = Context(; limit=0.4)
+                result = impute(m, DropObs(; context=ctx))
+                expected = m[[1, 4, 5], :]
+
+                @test isequal(result, expected)
+                @test isequal(result, Impute.dropobs(m; context=ctx))
+                @test isequal(collect(result'), Impute.dropobs(collect(m'); dims=2, context=ctx))
+
+                m_ = Impute.dropobs!(m; context=ctx)
+                # The mutating test is broken because we need to making a copy of
+                # the original matrix
+                @test_broken isequal(m, expected)
+                @test isequal(m_, expected)
+            end
+
+            @testset "Tables" begin
+                ctx = Context(; limit=0.4)
+                @testset "DataFrame" begin
+                    df = deepcopy(table)
+                    result = impute(df, DropObs(; context=ctx))
+                    expected = dropmissing(df)
+
+                    @test isequal(result, expected)
+                    @test isequal(result, Impute.dropobs(df; context=ctx))
+
+                    df_ = Impute.dropobs!(df; context=ctx)
+                    # The mutating test is broken because we need to making a copy of
+                    # the original table
+                    @test_broken isequal(df, expected)
+                    @test isequal(df_, expected)
+                end
+
+                @testset "Column Table" begin
+                    coltab = Tables.columntable(table)
+
+                    result = impute(coltab, DropObs(; context=ctx))
+                    expected = Tables.columntable(dropmissing(table))
+
+                    @test isequal(result, expected)
+                    @test isequal(result, Impute.dropobs(coltab; context=ctx))
+
+                    coltab_ = Impute.dropobs!(coltab; context=ctx)
+                    # The mutating test is broken because we need to making a copy of
+                    # the original table
+                    @test_broken isequal(coltab, expected)
+                    @test isequal(coltab_, expected)
+                end
+
+                @testset "Row Table" begin
+                    rowtab = Tables.rowtable(table)
+                    result = impute(rowtab, DropObs(; context=ctx))
+                    expected = Tables.rowtable(dropmissing(table))
+
+                    @test isequal(result, expected)
+                    @test isequal(result, Impute.dropobs(rowtab; context=ctx))
+
+                    rowtab_ = Impute.dropobs!(rowtab; context=ctx)
+                    # The mutating test is broken because we need to making a copy of
+                    # the original table
+                    # @test_broken isequal(rowtab, expected)
```

Comment:
```suggestion
                    @test_broken isequal(rowtab, expected)
```

Reply:
Fun fact, this test isn't broken :D

Reply:
can you please add a comment on that? (I didn't understand from the current comment why this test was commented out)